board/piksiv3/rootfs-overlay/etc/init.d/S38usb_gadget

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+name="usb_gadget"
+
+VENDOR_ID="0x0000"
+PRODUCT_ID="0x0000"
```

Comment:
I tried this as-is and the device was not usable on Windows (it was on Linux).

I changed the vendor ID and product ID and I was able to use it on Windows, Linux and MAC.